public/app/features/alerting/unified/components/rule-editor/PreviewRule.tsx

```diff
@@ -16,7 +16,7 @@ export function PreviewRule(): React.ReactElement | null {
   const styles = useStyles2(getStyles);
   const [preview, onPreview] = usePreview();
   const { getValues } = useFormContext();
-  const [type] = getValues(fields);
+  const { type, condition } = getValues();
```

Comment:
You should normally use `watch` instead of `getValues` in a FC to not re-render this particular value did not change